include/fiction/algorithms/simulation/sidb/maximum_defect_influence_distance.hpp

```diff
@@ -0,0 +1,304 @@
+//
+// Created by Jan Drewniok on 21.06.23.
+//
+
+#ifndef FICTION_MAXIMUM_DEFECT_INFLUENCE_DISTANCE_HPP
+#define FICTION_MAXIMUM_DEFECT_INFLUENCE_DISTANCE_HPP
+
+#include "fiction/algorithms/simulation/sidb/critical_temperature.hpp"
+#include "fiction/algorithms/simulation/sidb/quickexact.hpp"
+#include "fiction/layouts/bounding_box.hpp"
+#include "fiction/technology/sidb_defects.hpp"
+#include "fiction/technology/sidb_surface.hpp"
+#include "fiction/utils/execution_utils.hpp"
+#include "fiction/utils/layout_utils.hpp"
+
+#include <algorithm>
+#include <cmath>
+#include <limits>
+#include <utility>
+#include <vector>
+
+namespace fiction
+{
+/**
+ * This struct stores the parameters for the `maximum_defect_influence_distance` algorithm.
+ */
+template <typename Lyt>
+struct maximum_defect_influence_distance_params
+{
+    /**
+     * The defect used to calculate the maximum defect influence distance.
+     */
+    sidb_defect defect{};
+    /**
+     * Physical simulation parameters.
+     */
+    sidb_simulation_parameters physical_params{};
+    /**
+     * The pair describes the width and height of the area around the gate, which is
+     * also used to place defects (given in SiQAD coordinates).
+     */
+    std::pair<int32_t, int32_t> additional_scanning_area{50, 6};
+};
+
+/**
+ * This struct stores the `Maximum defect influence distance` and the corresponding `Maximum defect position` found by
+ * the algorithm.
+ */
```

Comment:
Its name suggests it stores the statistics gathered while running this algorithm, not its return type.